alerts/jetty/README.md

```diff
@@ -0,0 +1,42 @@
+# Alerts for Jetty in the Ops Agent
```

Comment:
Could we please add a description of what does this alert policy do and why is it important? 

Similar to this README https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/156/files#diff-7a1067bb99e3087c641d43fcc10cda016f5725e09ab8050c46fab4a9cd3e761d

Reply:
Oh, I see it down below on line 32. How about we move higher to first explain what this does, and then we can share the implementation details

Reply:
Updated